client/extensions/woocommerce/woocommerce-services/lib/initialize-carrier-accounts-state.js

```diff
@@ -0,0 +1,42 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+export default () => {
+	const requiredFields = [
+		'account_number',
+		'name',
+		'street1',
+		'city',
+		'state',
+		'country',
+		'postal_code',
+		'phone',
+		'email',
+		'name',
+		'title',
+		'website',
+		'license_agreement',
```

Comment:
Do we still need `license_agreement`? I don't see this used in the API post payload spec.

Reply:
It was needed in the form (it was a required field) but it isn't in the API, so I was removing it in the action that sends the form: https://github.com/Automattic/woocommerce-services/blob/5d87e9e2e22d6feddde09b02698e3dd30bac0d77/client/extensions/woocommerce/woocommerce-services/state/carrier-accounts/actions.js#L71

But we've removed the field in the form, so I'll get rid of all the references of this field.